src/session.ts

```diff
@@ -808,7 +808,7 @@ export class SessionManager implements Middleware {
 
             new SessionMenuItem(
                 "Modify 'powerShell.powerShellAdditionalExePaths' in Settings",
-                () => { vscode.commands.executeCommand("workbench.action.openSettingsJson"); }),
+                () => { vscode.commands.executeCommand("workbench.action.openSettings", "powershell.powerShellAdditionalExePaths"); }),
```

Comment:
if I change this to 
```suggestion
                () => { vscode.commands.executeCommand("workbench.action.openSettings", "powerShellAdditionalExePaths"); }),
```
Then I get the additionalExePaths AND the Default Version:
![image](https://user-images.githubusercontent.com/2644648/163108072-1448b5d8-2c7c-4a48-94ca-49e79692098f.png)

Which is prettttttty nice. Thoughts on making this change? The downside is that it won't be as explicit. This only works because the string is found in both settings.

Reply:
Might be a good opportunity to change the quick pick item text in the session menu too. Any suggestions?

Reply:
I like it!